cfn-template/deeplearning.template

```diff
@@ -84,6 +143,11 @@
        "AllowedValues" : [ "AmazonLinux", "Ubuntu" ],
        "ConstraintDescription" : "Amazon Supported Image Type"
      },
+    "AMIOverride" : {
```

Comment:
This is a really great feature that allow users not limited to current version of dlami. Thanks!